composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
```

Comment:
Oh, ah. Didn't release you make the version range bigger. Let's use `5.0 - 5.2`, that looks pretty clear :)